Makefile

```diff
@@ -415,3 +415,16 @@ test-app-dev: tmp/.ekam-run
 	@cp src/sandstorm/test-app/test-app.capnp tmp/sandstorm/test-app/test-app.capnp
 	@cp src/sandstorm/test-app/*.html tmp/sandstorm/test-app
 	spk dev -Isrc -Itmp -ptmp/sandstorm/test-app/test-app.capnp:pkgdef
+
+# ====================================================================
+# meteor-testapp.spk
+
+meteor-spk-0.3.2/meteor-spk:
+	@$(call color,downloading meteor-spk)
+	@curl https://dl.sandstorm.io/meteor-spk-0.3.2.tar.xz | tar Jxf -
+
+tests/assets/meteor-testapp.spk: meteor-testapp meteor-spk-0.3.2/meteor-spk $(shell find meteor-testapp -type f -o -type d)
+	@cd meteor-testapp && PATH="$$PATH:$$PWD/bin" && ../meteor-spk-0.3.2/meteor-spk pack ../tests/assets/meteor-testapp.spk
```

Comment:
From `spk help` it looks like the command should be:
> ../meteor-spk-0.3.2/meteor-spk -I../src pack ../tests/assets/meteor-testapp.spk

... but the above change yields this error:
> No file found to satisfy requirement: 
Makefile:430: recipe for target 'tests/assets/meteor-testapp.spk' failed
make: *** [tests/assets/meteor-testapp.spk] Error 1

Based on the discussion in [293](https://github.com/sandstorm-io/sandstorm/issues/293), I ran `meteor-spk dev` to try to get rid of that error, and got a new error:
> *** Uncaught exception ***
sandstorm/spk.c++:1115: failed: lstat(target.cStr(), &stats): No such file or directory; target = /home/bill/sandstorm/bin/bin/sandstorm-http-bridge
stack: 0x5503a8 0x54e593 0x54d6d1 0x54d60d 0x4f9041 0x4f9cc5
Makefile:430: recipe for target 'tests/assets/meteor-testapp.spk' failed
make: *** [tests/assets/meteor-testapp.spk] Error 1